plugindocs.rb

```diff
@@ -0,0 +1,56 @@
+require "clamp"
+require "json"
+require "fileutils"
+require "net/http"
+
+class PluginDocs < Clamp::Command
+  option "--output-path", "OUTPUT", "Path to the top-level of the logstash-docs path to write the output.", required: true
+  parameter "PLUGINS_JSON", "The path to the file containing plugin versions json"
+
+  def execute
+    report = JSON.parse(File.read(plugins_json))
+    plugins = report["successful"]
+
+    skip = [ "logstash-core-plugin-api", "logstash-patterns-core", "logstash-devutils" ]
+    skip += [ 
+      "logstash-codec-sflow", # Empty plugin repository
+      "logstash-filter-math", # Empty plugin repository
+      "logstash-input-mongodb", # Empty plugin repository
+      "logstash-filter-kubernetes_metadata", # https://github.com/logstash-plugins/logstash-filter-kubernetes_metadata/issues/2
+      "logstash-input-eventlog", # https://github.com/logstash-plugins/logstash-input-eventlog/issues/38
+    ]
```

Comment:
I've created https://github.com/logstash-plugins/logstash-input-eventlog/pull/39, once its merged we can remove this ignore.